helm/values.yaml

```diff
@@ -1,14 +1,22 @@
 environment: development
 postgresUsername: postgres@mine-support2
 postgresPassword: changeme
+postgresDatabase: mine_claims
```

Comment:
As it is the deployment is failing as the container and registry values are not being set correctly. 
The pipeline yaml is expecting the charts values to be named as per the standards https://eaflood.atlassian.net/wiki/spaces/FFCPD/pages/1303969817/FFC+Standard+Helm+Charts+Templates+and+Skaffold+Config

The user service is a good example of the structure, typically the only thing that needs to change are the env vars and image names.